test/functional/apps/management/data_views/_index_pattern_filter.ts

```diff
@@ -24,13 +24,16 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.settings.clickKibanaIndexPatterns();
     });
 
+    after(async function () {
+      await kibanaServer.savedObjects.cleanStandardList();
+    });
```

Comment:
Move it here, because it was the source of a race condition, because in a flaky way, the index pattern was already removed, but the cleanStandardList tried to remove it a second time